Src/WitsmlExplorer.Frontend/components/ContentViews/CurveValuesView.tsx

```diff
@@ -609,8 +614,16 @@ export const CurveValuesView = (): React.ReactElement => {
             overrideEndIndex={autoRefresh ? getCurrentMaxIndex() : null}
             onClickRefresh={() => refreshData()}
           />
-          <Switch checked={showPlot} onChange={() => setShowPlot(!showPlot)} />
-          <Typography>Show Plot</Typography>
+          <EdsProvider density={theme}>
```

Comment:
Is the provider used? Doesn't look like the child components depends on the EdsProvider.

Reply:
The EDS documentation instructs us to wrap it within an EDSProvider: [https://storybook.eds.equinor.com/?path=/docs/inputs-selection-controls-switch--docs#compact](https://storybook.eds.equinor.com/?path=/docs/inputs-selection-controls-switch--docs#compact)